src/main/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapper.java

```diff
@@ -2344,18 +2344,19 @@ private void buildMiscellaneousDJEvent(EventHistory.EventHistoryBuilder builder,
             .getLabel().startsWith("Both");
         String miscTextRequested =  "RPA Reason: Default Judgment requested and claim moved offline.";
         String miscTextGranted = "RPA Reason: Default Judgment granted and claim moved offline.";
-
+        LocalDateTime eventDJCreatedDate = LocalDateTime.now();
         if (featureToggleService.isJOLiveFeedActive()
             && caseData.getCcdState() == CaseState.All_FINAL_ORDERS_ISSUED) {
```

Comment:
I think this conidition "&& caseData.getCcdState() == CaseState.All_FINAL_ORDERS_ISSUED" will modify events misceleneous text and also break event sequence if the case is taken offline. May be we need to remove this condition.